Files/Strings/en-US/Resources.resw

```diff
@@ -2779,7 +2779,13 @@ We use App Center to keep track of app usage, find bugs, and fix crashes. All in
   <data name="SettingsImportErrorTitle" xml:space="preserve">
     <value>Error importing settings</value>
   </data>
-   <data name="Sidebar" xml:space="preserve">
+  <data name="Empty" xml:space="preserve">
+    <value>Empty</value>
+  </data>
+  <data name="EmptyRecycleBin" xml:space="preserve">
+    <value>Empty Recycle Bin</value>
+  </data>
```

Comment:
Should we merge these and use just one or the other?

Reply:
We can use "Empty Recycle Bin" for the 2. But in this case why put a tooltip?

Reply:
Let's keep it how it is, good work!

Reply:
After thinking it over, it's better to put the long label everywhere. This avoids having to know the context. It is modified shortly before your message.